gapir/cc/interpreter.cpp

```diff
@@ -123,6 +123,9 @@ bool Interpreter::run(const uint32_t* instructions, uint32_t count) {
 
 void Interpreter::exec() {
   for (; mCurrentInstruction < mInstructionCount; mCurrentInstruction++) {
+    if (mCurrentInstruction % 1000 == 0) {
+      GAPID_WARNING("%u/%u", mCurrentInstruction, mInstructionCount);
```

Comment:
Extra debug message

Reply:
Removed. Good catch! Sorry for all the debug prints left in the first version of this pull request. I forgot to strip them before uploading!